### PR TITLE
add snax-alu to dockerfile

### PR DESCRIPTION
**Pull request by jorendumoulin**
for compiler work on the streamers!

### CODE REVIEW

File: util/container/Dockerfile [hunk 1]
```diff
@@ -129,6 +129,21 @@ RUN git clone https://github.com/kuleuven-micas/snitch_cluster /src && \
     make -C target/snitch_cluster bin/snitch_cluster.vlt \
     CFG_OVERRIDE=cfg/snax-gemm.hjson -j $(nproc)
 
+FROM base as snax-alu
+
+RUN git clone https://github.com/kuleuven-micas/snitch_cluster /src && \
```
Comment (JosseVanDelm) [inline util/container/Dockerfile:134]: Could you rename the other ones too?
```suggestion
RUN git clone https://github.com/kuleuven-micas/snax_cluster /src && \
```

File: util/container/Dockerfile [hunk 2]
```diff
@@ -151,3 +166,13 @@ COPY --from=snax-gemm /src/target/snitch_cluster/sw/snax/ /opt/snax-gemm/target/
...
+
+COPY --from=snax-alu /src/target/snitch_cluster/bin/snitch_cluster.vlt /opt/snax-alu-rtl/bin/snitch_cluster.vlt
+COPY --from=snax-alu /src/sw/snRuntime /opt/snax-alu/sw/snRuntime
+COPY --from=snax-alu /src/target/snitch_cluster/sw/runtime/rtl /opt/snax-alu/target/snitch_cluster/sw/runtime/rtl
+COPY --from=snax-alu /src/target/snitch_cluster/sw/runtime/rtl-generic /opt/snax-alu/target/snitch_cluster/sw/runtime/rtl-generic
+COPY --from=snax-alu /src/target/snitch_cluster/sw/runtime/common /opt/snax-alu/target/snitch_cluster/sw/runtime/common
+COPY --from=snax-alu /src/target/snitch_cluster/sw/snax/ /opt/snax-alu/target/snitch_cluster/sw/snax
+COPY --from=snax-alu /src/sw/math/ /opt/snax-alu/sw/math/
+COPY --from=snax-alu /src/sw/deps/riscv-opcodes /opt/snax-alu/sw/deps/riscv-opcodes
+COPY --from=snax-alu /src/sw/deps/printf /opt/snax-alu/sw/deps/printf
```
Comment (JosseVanDelm) [inline util/container/Dockerfile:178]: maybe you could also do this:
https://stackoverflow.com/questions/30256386/how-to-copy-multiple-files-in-one-layer-using-a-dockerfile

**Review by JosseVanDelm — APPROVED** [review]
looking good

File: util/container/Dockerfile (hunk 2 above)
Comment (jorendumoulin) [reply util/container/Dockerfile:178]: It doesn't seem straightforward with all the nested folders going to different destinations. If ok I'll ignore this as I wanna move fast today, but I'll look into this maybe later (maybe we can just remove all the copies as im not sure we need all the software duplicated in the docker)

**Review by rgantonio — APPROVED** [review]
Need another cookie! Hihi

**Pull request merged**